src/main/java/com/hotels/bdp/circustrain/bigquery/partition/HivePartitionLocationConverter.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.bigquery.partition;
+
+import static com.google.api.client.repackaged.com.google.common.base.Preconditions.checkNotNull;
+
+import com.hotels.bdp.circustrain.bigquery.extraction.container.ExtractionUri;
+
+class HivePartitionLocationConverter {
+
+  private final String extractionLocation;
+
+  HivePartitionLocationConverter(ExtractionUri extractionUri) {
+    checkNotNull(extractionUri, "extractionUri must not be null");
+    this.extractionLocation = "gs://" + extractionUri.getBucket() + "/" + extractionUri.getFolder() + "/";
```

Comment:
This looks quite similar to the `HiveTableLocationFactory` class above, do we need it?

Reply:
Isn't the above now just `ExtractionUri.getTableLocation()`? Also, if we have that method do we even really need this class?